e2e/process-services-cloud/task-form-cloud-component.e2e.ts

```diff
@@ -258,4 +295,187 @@ describe('Task form cloud component', () => {
         });
     });
 
+    describe('Complete task - cloud directive', () => {
+
+        beforeEach(async () => {
+            await navigationBarPage.navigateToProcessServicesCloudPage();
+            await appListCloudComponent.checkApsContainer();
+            await appListCloudComponent.goToApp(simpleApp);
+        });
+
+        it('[C315174] Should be able to complete a standalone task with visible tab with empty value for field', async () => {
+            await tasksCloudDemoPage.myTasksFilter().clickTaskFilter();
+            await expect(await tasksCloudDemoPage.getActiveFilterName()).toBe('My Tasks');
+
+            await tasksCloudDemoPage.taskListCloudComponent().checkContentIsDisplayedByName(visibilityConditionTasks[0].entry.name);
+            await tasksCloudDemoPage.taskListCloudComponent().selectRow(visibilityConditionTasks[0].entry.name);
+            await taskHeaderCloudPage.checkTaskPropertyListIsDisplayed();
+
+            await widget.tab().checkTabIsDisplayedByLabel(tab.tabWithFields);
+            await widget.tab().checkTabIsNotDisplayedByLabel(tab.tabFieldValue);
+            await widget.textWidget().isWidgetVisible(widgets.textOneId);
+            await widget.textWidget().isWidgetNotVisible(widgets.textTwoId);
+
+            await widget.textWidget().setValue(widgets.textOneId, value.displayTab);
+            await widget.tab().checkTabIsDisplayedByLabel(tab.tabWithFields);
+            await widget.tab().checkTabIsDisplayedByLabel(tab.tabFieldValue);
+
+            await taskFormCloudComponent.clickCompleteButton();
+            await expect(await tasksCloudDemoPage.getActiveFilterName()).toBe('My Tasks');
+            await tasksCloudDemoPage.taskListCloudComponent().checkContentIsNotDisplayedByName(visibilityConditionTasks[0].entry.name);
+
+            await tasksCloudDemoPage.completedTasksFilter().clickTaskFilter();
+            await tasksCloudDemoPage.taskListCloudComponent().checkContentIsDisplayedByName(visibilityConditionTasks[0].entry.name);
+            await tasksCloudDemoPage.taskListCloudComponent().selectRow(visibilityConditionTasks[0].entry.name);
+            await widget.tab().checkTabIsDisplayedByLabel(tab.tabWithFields);
+            await widget.tab().checkTabIsDisplayedByLabel(tab.tabFieldValue);
+        });
+
+        it('[C315177] Should be able to complete a standalone task with invisible tab with invalid value for field', async () => {
+            // ACTIVITI-3746
```

Comment:
do we need this comment?

Reply:
no, i will remove the comment when that bug will be fixed. Currently that bug is in PR ready, so hopefully it will not take too much until then.